### ISSUE TITLE
Is there a handy Hadoop cluster running?

### ISSUE THREAD

**Issue by biolearning** [amplab/spark-ec2#70]
With Spark cluster launched and started with ec2 script, is there the Hadoop cluster ready to go, or just an easy command to start it? The reason I am asking for this is - I would like to enable log and save that in hadoop FS which is accessible for Spark workers?

**Comment by shivaram** [comment]
Yes - there should be a HDFS installation on `/root/ephemeral-hdfs`

**Comment by biolearning** [comment]
Thanks for your reply, @shivaram, is it started by default? I guess not, then what is the command I can start the Hadoop cluster?

**Comment by shivaram** [comment]
It should be started by default. If you want to reconfigure / restart it you can read the Hadoop docs for the appropriate version.
